bundle/keys.go

```diff
@@ -88,12 +89,23 @@ func NewSigningConfig(key, alg, claimsPath string) *SigningConfig {
 	}
 
 	return &SigningConfig{
+		Plugin:     defaultSignerID,
 		Key:        key,
 		Algorithm:  alg,
 		ClaimsPath: claimsPath,
 	}
 }
 
+// WithPlugin sets the signing plugin in the signing config
+func (s *SigningConfig) WithPlugin(plugin string) *SigningConfig {
+	if plugin == "" {
+		s.Plugin = defaultSignerID
+	} else {
+		s.Plugin = plugin
+	}
```

Comment:
I think that works...I originally had it like this because I wasn't setting the defaultSignerID in NewSigningConfig, but with that there it makes sense to remove this.